pwiz_tools/Skyline/Controls/Graphs/GraphFullScan.cs

```diff
@@ -726,6 +743,14 @@ private void  CreateSingleScan()
                     var item = new SpectrumItem(pointLists[i], GetTransitionColor(transition), _msDataFileScanHelper.ScanProvider.Transitions[i].Name, 2);
                     var curveItem = _graphHelper.GraphControl.AddGraphItem(GraphPane, item, false);
                     curveItem.Label.IsVisible = false;
+                    if (massErrors != null)
+                    {
+                        var errorAccumulator =
+                            new SpectrumFilterPair.MeanErrorAccumulator(true, ChromExtractor.summed,
+                                transition.ProductMz);
+                        pointLists[i].ForEach(pt => errorAccumulator.AddPoint(pt.X, pt.Y));

```

Comment:
I think you need to go back to the original spectrum in order to know all of the mz's and intensities that are within this transition's extraction window.
pointLists[i] is going to be missing some values if the extraction windows for the transitions overlap with each other

Those pointLists arrays are appropriate for deciding what color to make the lines on the graph, but for actually calculating the mass error I think you need to go back to the original spectrum.